.appveyor.yml

```diff
@@ -33,10 +34,10 @@ build_script:
     -DLAPACK_LIBRARY:FILEPATH=%BLAS_LIBRARY%
     -DCMAKE_PREFIX:FILEPATH="%APPVEYOR_BUILD_FOLDER%/armadillo"
     -DBUILD_SHARED_LIBS=OFF
-    -DCMAKE_BUILD_TYPE=Release ..
+    -DCMAKE_BUILD_TYPE=Debug ..
   - >
     "%MSBUILD%" "armadillo.sln"
-    /m /verbosity:quiet /p:Configuration=Release;Platform=x64
+    /m /verbosity:quiet /p:Configuration=Debug;Platform=x64
```

Comment:
We could add `/std:c++11` (https://docs.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version?view=msvc-160) and see if that helps.

Reply:
Good ol' Windows.